### PR TITLE
Add support for BF SDE 9.6.0

### PR DESCRIPTION
**Pull request by bocon13**
SDE 9.6.0 has changed the build process. `p4studio_build.py` is now `p4studio`.

`stratum_bf` and `stratum_bfrt` start up cleanly against `tofino-model`

Notable changes
- `libbf_switchd_lib.a` no longer exists and required symbols appear to be in `libdriver.so`
- The `libpiall` dependency transitioned from a static library to shared library
- `libJudy` transitioned from a static library to shared library

TODO
- [ ] conditionally depend on `libbf_switchd_lib.a` based on SDE version
- [ ] verify the dependencies are being included

This PR doesn't address the build script changes required for 9.6.0; you need to build `bf-sde-9.6.0-install.tgz` by hand with a few patches to the CMake files. Follow #795 for details on that.

### CODE REVIEW

File: stratum/hal/bin/barefoot/docker/start-stratum-container.sh [hunk 1]
```diff
@@ -5,7 +5,8 @@ set -e
 
 LOG_DIR=${LOG_DIR:-/var/log}
 SDE_VERSION=${SDE_VERSION:-9.3.2}
-DOCKER_IMAGE=${DOCKER_IMAGE:-stratumproject/stratum-bf}
+#FIXME update "stratum-bf" to "stratum-bfrt" globally
```
Comment (bocon13) [inline stratum/hal/bin/barefoot/docker/start-stratum-container.sh:8]: make this a separate PR

Comment (pudelkoM) [reply stratum/hal/bin/barefoot/docker/start-stratum-container.sh:8]: Done: #789

**Comment by Yi-Tseng** [comment]
there is  a `libbfrt.so` that includes bfrt symbols we are usually using, I guess we also need to include it?

File: stratum/hal/bin/barefoot/docker/start-stratum-container.sh (hunk 1 above)
Comment (smittal-bf) [reply stratum/hal/bin/barefoot/docker/start-stratum-container.sh:8]: Thanks Max and Brian for this quick support!

Comment (bocon13) [reply stratum/hal/bin/barefoot/docker/start-stratum-container.sh:8]: thanks!

**Comment by bocon13** [comment]
We are skipping SDE 9.6.0
